CHANGELOG.md

```diff
@@ -282,7 +289,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Initial implementation
 
-[Unreleased]: https://github.com/owncloud/files_antivirus/compare/v0.15.0...master
+[Unreleased]: https://github.com/owncloud/files_antivirus/compare/v0.15.0...v0.15.1
```

Comment:
```suggestion
[Unreleased]: https://github.com/owncloud/files_antivirus/compare/v0.15.0...master
```

This should do a diff up to `master` head?

Reply:
Instead of `master` I recommend to use `HEAD` - Using `HEAD` will always point to the latest changes on the default branch ;-)